fluent-plugin-enhance-k8s-metadata/lib/sumologic/kubernetes/connector.rb

```diff
@@ -41,6 +41,7 @@ def create_client(base, ver)
         client
       rescue StandardError => e
         log.error e
+        nil
```

Comment:
Shouldn't we retry/fail here? Does it make sense to proceed if the client is not created? 😕

Reply:
Yeah, thinking about it you are absolutely right. I’ll make the change, thanks Dominik